client/src/components/Ratings/ReviewList.jsx

```diff
@@ -81,5 +133,7 @@ const Botton = styled.button`
   padding: 1.3rem 1rem 1.3rem 1rem;
   font-size: medium;
   font-weight: 700;
-
+  &:hover {
+    background-color:${(props) => props.theme.colors.buttonHover}
```

Comment:
Recently learned you can destructure this a little and do `${({ theme }) => theme.colors ....`. It's still annoying to type but it helps a bit.

Reply:
Nice!! Thanks for sharing